### DISCUSSION TITLE
Statusmap and Annotations not Working?

### DISCUSSION THREAD

**Discussion by Jtimann** [flant/grafana-statusmap#255] [General]
Hello,

i created a Dashboard which shows Tests and Deployments.
For the Tests i created a statusmap for positive and negative tests. This all gone good.

In the past i added the deployments over the annotations so i can see if a deployment is done, how the tests react.
This worked a few months. But nowadays the annotations don't show up in the statusmap-panel. If a create normal panel, so no statusmap the annotations show up.
Is there a bug in a newer version?
Currently i am at Grafana version 8.3.4 
On the following picture u see the statusmap and the normal panel. The yellow annotations only show up on the normal one.

![MicrosoftTeams-image](https://user-images.githubusercontent.com/103050723/161773246-8150337a-e3b3-418e-b052-1cfdef7479f6.png)

Thanks for your reply.

**Comment by diafour** [comment]
Hello! Annotations support in Angular panels is somewhat hacky, so new Grafan version can break it. What datasource you use for annotations?

**Comment by Jtimann** [comment]
Hello @diafour,
we updated Grafana to Version v8.5.2 and the Statusmap plugin is on Version 0.5.1.
The Database is InfluxDB version 1.8.10.

After the Grafana Update we still have the Problem that the annotations won't show up in the statusmap.
Same Dashboard other panel e.g. time series with the same select query the annotations load.

U have any idea what causes this problem?

Thank you for your help.

**Comment by chkp-lidorta** [comment]
Having the same issue using Grafana v10.0.12 , other panel types work fine
